ugwumadu116/Auto-Mart-Andela
# #166115604 build out user(admin) can delete car

#### What does this PR do?
- Creates a route to delete car AD
#### Description of Task to be completed?
- Create a route that uses DELETE/car/car_id eg `DELETE/api/v1/car/1`
- Create a car service to handle all car functionalities
- Create a car controller to handle route request
- Create a car validator to know the user that is making the request
#### How should this be manually tested?
- Login in with the your email and password 
  - example `{
	"email": admin@gmail.com",
	"password": "adminPwd"
}`
- Use your login token and set it in Headers and set the key to be authorization while the value will be your token
- Then make a DELETE request to `/api/v1/car/1`
- Your messages will appear in your response
#### What are the relevant pivotal tracker stories?
#166115604
